<!-- to be launched in rpi -->
<!-- Launch the mavros instance that connects with drone, run the node, lora_bridge
    launch rosserial, run server_application.py in that order 
-->

<launch>
    <include file="$(find px4_controller)/launch/px4_sitl.launch"></include>
    <node pkg="rosserial_python" type="serial_node.py" name="rosserial_python" output="screen">
        <param name="port" value="/dev/ttyACM0"/>
        <param name="baud" value="57600"/>
    </node>
    <node pkg="px4_controller" type="lora_bridge.py" name="lora_bridge"></node>
</launch>